enhancements/update/cvo-log-level-api.md

```diff
@@ -0,0 +1,213 @@
+---
+title: cvo-log-level-api
+authors:
+  - "@Davoska"
+reviewers:
+  - "@wking"
+  - "@LalatenduMohanty"
+  - "@petr-muller"
+approvers:
+  - "@LalatenduMohanty"
+api-approvers:
+  - "@deads2k"
+  - "@bparees"
+creation-date: 2023-10-09
+last-updated: 2023-11-20
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-1029
+---
+
+# CVO Log level API
+
+## Summary
+
+This enhancement describes the API changes needed to provide a simple way of dynamically changing the Cluster Version Operator's log level.
+
+## Motivation
+
+The Cluster Version Operator (CVO) logs a lot of useful information. However, there is currently no way to easily change the log level for the CVO.
+
+It would be useful to provide functionality for the cluster administrators and OpenShift engineers to easily modify the log level to a desired level.
```

Comment:
This feels like the sort of thing other cluster operators might benefit from, too. Do they already have their own implementation for managing the log level?

Reply:
Hello! I do wholeheartedly apologize for not originally responding.

Yes, a cluster administrator may modify the log level of cluster operators that are represented by `ClusterOperator` Custom Resources, as the `ClusterOperatorSpec`  includes a `logLevel` field (see https://github.com/openshift/api/blob/3e5de946111c67cb1f2b09ff7da569012d15931d/operator/v1/types.go#L58-L65).